lib/honeybadger/plugins/delayed_job/plugin.rb

```diff
@@ -44,14 +44,19 @@ class Plugin < ::Delayed::Plugin
 
               block.call(job)
             rescue Exception => error
-              DelayedJobBadger.notify(
-                :component     => component,
-                :action        => action,
-                :error_class   => error.class.name,
-                :error_message => "#{ error.class.name }: #{ error.message }",
-                :backtrace     => error.backtrace,
-                :exception     => error
-              ) if job.attempts.to_i >= DelayedJobBadger.config[:'delayed_job.attempt_threshold'].to_i
+              acceptable_errors = job.payload_object.respond_to?(:acceptable_errors) ? job.payload_object.acceptable_errors : []
+              # Skip reporting temporarily failed jobs that have an acceptable exception
+              if job.attempts.to_i >= DelayedJobBadger.config[:'delayed_job.attempt_threshold'].to_i &&
```

Comment:
Quick question. Today when a job that has 3 attempts fails on the first attempt with a StandardError, we get a Honeybadger notification. Will this check, `job.attempts.to_i >= DelayedJobBadger.config[:'delayed_job.attempt_threshold'].to_i` avoid the notification?

Reply:
No, this check has to do with something else separate. I think this check skips notifying if a job errors over 25 times, which is the default setting for attempt threshold.

Normally, the job just fails up to max_attempt times, and each time this hook will send a honeybadger notification.